lte/gateway/python/integ_tests/s1aptests/test_attach_he_policy.py

```diff
@@ -0,0 +1,232 @@
+        )
+
+        print("Sleeping for 5 seconds")
+        time.sleep(5)
+        # Send PDN Connectivity Request
+        apn = "ims"
+        self._s1ap_wrapper.sendPdnConnectivityReq(ue_id, apn)
+        # Receive PDN CONN RSP/Activate default EPS bearer context request
+        response = self._s1ap_wrapper.s1_util.get_response()
+        self.assertEqual(
+            response.msg_type, s1ap_types.tfwCmd.UE_PDN_CONN_RSP_IND.value
+        )
+        act_def_bearer_req = response.cast(s1ap_types.uePdnConRsp_t)
+        addr = act_def_bearer_req.m.pdnInfo.pAddr.addrInfo
+        sec_ip = ipaddress.ip_address(bytes(addr[:4]))
+
+        print(
+            "********************** Sending Activate default EPS bearer "
+            "context accept for UE id ",
+            ue_id,
+        )
+
+        print("Sleeping for 5 seconds")
+        time.sleep(5)
+
+        dl_flow_rules = {
+            default_ip: [flow_list1],
+        }
+        # 1 UL flow is created per bearer
+        num_ul_flows = 2
+        # Verify if flow rules are created
+        self._s1ap_wrapper.s1_util.verify_flow_rules(
+            num_ul_flows, dl_flow_rules
+        )
+        print(
+            "************* Sending UE context release request ",
+            "for UE id ",
+            ue_id,
+        )
+        # Send UE context release request to move UE to idle mode
+        req = s1ap_types.ueCntxtRelReq_t()
+        req.ue_Id = ue_id
+        req.cause.causeVal = gpp_types.CauseRadioNetwork.USER_INACTIVITY.value
+        self._s1ap_wrapper.s1_util.issue_cmd(
+            s1ap_types.tfwCmd.UE_CNTXT_REL_REQUEST, req
+        )
+        response = self._s1ap_wrapper.s1_util.get_response()
+        self.assertEqual(
+            response.msg_type, s1ap_types.tfwCmd.UE_CTX_REL_IND.value
+        )
+
+        print("Sleeping for 5 seconds")
+        time.sleep(5)
+
+        # Verify if paging flow rules are created
+        ip_list = [default_ip, sec_ip]
+        self._s1ap_wrapper.s1_util.verify_paging_flow_rules(ip_list)
+
+        print(
+            "************************* Sending Service request for UE id ",
+            ue_id,
+        )
+        # Send service request to reconnect UE
+        req = s1ap_types.ueserviceReq_t()
+        req.ue_Id = ue_id
+        req.ueMtmsi = s1ap_types.ueMtmsi_t()
+        req.ueMtmsi.pres = False
+        req.rrcCause = s1ap_types.Rrc_Cause.TFW_MO_SIGNALLING.value
+        self._s1ap_wrapper.s1_util.issue_cmd(
+            s1ap_types.tfwCmd.UE_SERVICE_REQUEST, req
+        )
+        response = self._s1ap_wrapper.s1_util.get_response()
+        self.assertEqual(
+            response.msg_type, s1ap_types.tfwCmd.INT_CTX_SETUP_IND.value
+        )
+
+        print("Sleeping for 5 seconds")
+        time.sleep(5)
+
+        # Verify if flow rules are created
+        self._s1ap_wrapper.s1_util.verify_flow_rules(
+            num_ul_flows, dl_flow_rules
+        )
+
```

Comment:
can be removed if idle/active mode transition is to be eliminated. for this test, looks redundant...